klangbecken.liq

```diff
@@ -65,7 +72,9 @@ def check_next_func(r) =
   filename = request.filename(r)
 
   last_play = float_of_string(metadata["last_play_epoch"])
-  if last_play == 0.0 then
+  if filename == "" then
+    false
```

Comment:
When is the filename empty? Is it because of flac issues you mentioned in #96?

Reply:
This happens from time to time, somewhat at random. I don't know why. It might disappear when updating to Liquidsoap 1.4?!?

Reply:
Liquidsoap 1.4 is coupled to CentOS 8 if we keep RPMs (and building your own ocaml toolchain on CentOS 7 isn't fun). I'll try again soonish anyway to see if we can get 1.4 available for CentOS 7 as well.

Reply:
For now Liquidsoap 1.3 is working fine. 

And I am all for moving to Liquidsoap 1.4 with an update to CentOS 8.